nvflare/app_common/metrics_exchange/metrics_sender.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any
+
+from nvflare.apis.analytix import AnalyticsDataType
+from nvflare.apis.event_type import EventType
+from nvflare.apis.fl_context import FLContext
+from nvflare.apis.utils.analytix_utils import create_analytic_dxo
+from nvflare.app_common.widgets.streaming import AnalyticsSender
+from nvflare.fuel.utils.constants import PipeChannelName
+from nvflare.fuel.utils.pipe.pipe import Message, Pipe
+from nvflare.fuel.utils.pipe.pipe_handler import PipeHandler
+
+
+class MetricsSender(AnalyticsSender):
```

Comment:
changes in this PR LGTM as a first stage.

We can think of these questions:
- Is `MetricsSender` a `AnalyticsSender`? should we inherit it? or should we use some composition? or should it be just `FLComponent`? or a `Widget`?